tests/test_article.py

```diff
@@ -196,21 +196,6 @@ def test_get_funder_identifier_fail(self):
         self.funding_award.institution_id = institution_id
         self.assertEqual(self.funding_award.get_funder_identifier(), None)
 
-        # Original:
-        self.assertRaises(ValueError, self.funding_award.get_funder_identifier())
```

Comment:
I removed this as I do not believe you need it as the above assertion handles the failure case of `get_funder_identifier`. (remove comments below detail the cases)

Reply:
This was probably just a test in order to get absolutely 100% coverage. That's not realistic so the test can be omitted.